src/main/java/com/conveyal/r5/rastercost/CustomCostField.java

```diff
@@ -122,8 +145,11 @@ public int additionalTraversalTimeSeconds (EdgeStore.Edge currentEdge, int baseT
         if (Double.isNaN(additionalCostSeconds)) {
             throw new CustomCostFieldException("Custom cost addition result is NaN for osmId:  " + currentEdge.getOSMID());
         }
+        int roudedAdditionalCostSeconds = (int) Math.round(additionalCostSeconds);
```

Comment:
the variable name has a typo I assume :)

Reply:
Good spotting, will fix all of this typos!